### PR TITLE
Make sure dh_clean not touch potential source files.

### PR DESCRIPTION
**Pull request by amosbird**
contrib/ssl/include/bio.h.orig.

I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

For changelog. Remove if this is non-significant change.

Category (leave one):
- Build/Testing/Packaging Improvement

### CODE REVIEW

**Comment by proller** [comment]
Why you want to change owner of etc/* directories?
Usually only root can change config files.
Making write access to "clickhouse" user is insecure.
Preprocessed configs was moved from etc to data dir witj this reason.

**Comment by amosbird** [comment]
> Usually only root can change config files.

I don't understand. Isn't `clickhouse` user supposed to be the dbadmin user? How come it's insecure when user `clickhouse` can rw data?

**Comment by alexey-milovidov** [comment]
Suppose we have some vulnerability in ClickHouse that allow us to write arbitrary files in filesystem. If the configs are writeable by clickhouse user, this will allow to change config using this vulnerability.

**Comment by amosbird** [comment]
OK, now I see that user `clickhouse` exists for dropping privileges. Perhaps we can apply it to the log directory too. nginx creates the log file before dropping privileges to `http` user so that the log folder is only writable to root.

**Comment by amosbird** [comment]
Yeah, it's easier to just have log dir belong to `clickhouse`

File: debian/rules [hunk 1]
```diff
@@ -92,7 +92,7 @@ override_dh_auto_test:
 
 override_dh_clean:
 	rm -rf debian/copyright debian/clickhouse-client.docs debian/clickhouse-common-static.docs
-	dh_clean
+	dh_clean -X contrib
```
Comment (alexey-milovidov) [inline debian/rules]: We can merge this line :)

File: debian/clickhouse-server.postinst [hunk 2]
```diff
@@ -101,9 +101,7 @@ Please fix this and reinstall this package." >&2
 
     if [ ! -d ${CLICKHOUSE_LOGDIR} ]; then
         mkdir -p ${CLICKHOUSE_LOGDIR}
-        chown root:${CLICKHOUSE_GROUP} ${CLICKHOUSE_LOGDIR}
-        # Allow everyone to read logs, root and clickhouse to read-write
-        chmod 775 ${CLICKHOUSE_LOGDIR}
+        chown ${CLICKHOUSE_USER}:${CLICKHOUSE_GROUP} ${CLICKHOUSE_LOGDIR}
```
Comment (alexey-milovidov) [inline debian/clickhouse-server.postinst]: But this modification is still questionable.

Comment (amosbird) [reply debian/clickhouse-server.postinst]: I don't see the difference of having 775 with root:clickhouse and 755 with clickhouse:clickhouse

Comment (alexey-milovidov) [reply debian/clickhouse-server.postinst]: It is important for somewhat reason.
It's unlikely that developer implemented it that way without some motivation (but they forgot to add comments).

Comment (amosbird) [reply debian/clickhouse-server.postinst]: ok.. I'll rollback it. Luckily there is still one line left to commit ;)

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**